extensions/amp-timeago/0.2/timeago.js

```diff
@@ -14,28 +14,39 @@
  * limitations under the License.
  */
 
-import {createElement, useRef} from '../../../src/preact';
+import {createElement, useEffect, useRef, useState} from '../../../src/preact';
 import {timeago} from '../../../third_party/timeagojs/timeago';
-import {useIsIntersecting} from '../../../src/preact/use-is-intersecting';
 import {useResourcesNotify} from '../../../src/preact/utils';
 
 /**
  * @param {!JsonObject} props
  * @return {Preact.Renderable}
  */
 export function Timeago(props) {
-  /** @type {{current: ?string}} */ const timestampRef = useRef(null);
+  const {1: setIsIntersecting} = useState(null);
```

Comment:
Array destructuring is merged, you just need to rebase.

```suggestion
  const [, setIsIntersecting] = useState(null);
```